ctx_test.go

```diff
@@ -5051,9 +5056,9 @@ func Test_Ctx_RepeatParserWithSameStruct(t *testing.T) {
 	defer app.ReleaseCtx(c)
 
 	type Request struct {
-		QueryParam  string `query:"query_param"`
-		HeaderParam string `reqHeader:"header_param"`
-		BodyParam   string `json:"body_param" xml:"body_param" form:"body_param"`
+		QueryParam  string `                                    query:"query_param"`
```

Comment:
Seems wrong spacing

Reply:
Actually, it's done intentionally so one can better differentiate the various tags being used.

Reply:
But the line width will increase. I'd prefer old layout